modules/presets/index.js

```diff
@@ -0,0 +1,5 @@
+export { Category } from './category.js';
+export { Collection } from './collection.js';
+export { Field } from './field.js';
+export { Preset } from './preset.js';
+export { presets } from './presets.js';
```

Comment:
as I mentioned in the PR description, `iD.presets` was itself a function; it also contained the other functions as properties, e.g. `iD.presets.Field`, etc.

I didn't think it was wise (or possible without being kludge-y) to carry that forward to the ES6 modules approach, so I moved it to the same level as the other functions... maybe I'm wrong though :)